lab-Marshall/README.md

```diff
@@ -0,0 +1,9 @@
+## Welcome to a simple IRC server!
+
+Getting started is a little clumsy but it's simple enough:
+
+To test functionality, please make your way in the terminal to lab-5-tcp-chat-server/lab-Marshall and run 'node server.js' to start up the server on port 3000.
+
+Open up two more terminals and run 'telnet localhost 3000' in each terminal to connect to the local server.
+
+Then, you should be able to begin testing chat functionality between the two local connections by typing anything you wish into the command line!
```

Comment:
Thanks for the readme! Looks good. I would add a step to instruct a user how to install your app, and how to disconnect (in your case, by typing 'END') from a session.